tests/cypress/e2e/Admin/Cookie.spec.ts

```diff
@@ -0,0 +1,168 @@
+        cy.reload();
+        cy.get('#cookie-banner').should('have.css', 'display', 'none');
+    });
+
+    it('Should hide banner and set cookie when rejecting analytics', () => {
+        cy.visit(Cypress.config().baseUrl ?? "");
+        
+        // Click reject and verify display state
+        cy.get('#reject-cookies').click();
+        cy.wait(1000);
+        cy.get('#cookie-banner').should('not.exist');
+        
+        // Verify cookie was set
+        cy.getCookie('analytics-cookies-consent').should('have.property', 'value', 'false');
+        
+        // Verify banner stays hidden on next visit
+        cy.reload();
+        cy.get('#cookie-banner').should('have.css', 'display', 'none');
+    });
+
+    it('Should initialize Clarity when analytics are accepted', () => {
+    cy.visit(Cypress.config().baseUrl ?? "");
+
+    // Accept cookies
+    cy.get('#accept-cookies').click();
+
+    // Wait for Clarity cookies to be created
+    cy.wait(5000);
+
+    // Check and log Clarity cookies
+    cy.getCookie('_clarity').then((cookie) => {
+        if (cookie) {
+            cy.log('_clarity cookie value:', cookie.value);
+        } else {
+            cy.log('_clarity cookie is missing.');
+        }
+    });
+
+    cy.getCookie('CLARITY_MASTERID').then((cookie) => {
+        if (cookie) {
+            cy.log('CLARITY_MASTERID cookie value:', cookie.value);
+        } else {
+            cy.log('CLARITY_MASTERID cookie is missing.');
+        }
+    });
+});
+
+    
+    
+
+    it('Should remove Clarity cookies when analytics are rejected', () => {
+        // First accept to set cookies
+        cy.visit(Cypress.config().baseUrl ?? "");
+        cy.get('#accept-cookies').click();
+        cy.wait(1000);
+        
+        // Then reject to remove them
+        cy.clearCookies();
+        cy.visit(Cypress.config().baseUrl ?? "");
+        cy.get('#reject-cookies').click();
+        cy.wait(1000);
+        
+        // Verify Clarity cookies were removed
+        cy.getCookie('_clarity').should('not.exist');
+        cy.getCookie('CLARITY_MASTERID').should('not.exist');
+
+        // Verify Clarity script is not added
+        cy.window().then((win) => {
+            const clarityId = win.document.body.getAttribute('data-clarity');
+            cy.get(`script[src*="clarity.ms/tag/${clarityId}"]`)
+                .should('not.exist');
+        });
+    });
+
+    it('Should maintain cookie choice across sessions', () => {
+        // Visit the page and accept cookies
+        cy.visit(Cypress.config().baseUrl ?? "");
+        cy.get('#accept-cookies').click();
+    
+        // Reload the page to confirm the cookie banner is hidden
+        cy.reload();
+        cy.get('#cookie-banner', { timeout: 10000 }).should('have.css', 'display', 'none');
+    
+        // Verify the cookie was set correctly
+        cy.getCookie('analytics-cookies-consent').should('exist').and('have.property', 'value', 'true');
```

Comment:
I wouldn't care about the cookie being set or not in cypress, that's implementation details